README.md

```diff
@@ -39,16 +39,98 @@ BibTeX entry:
 }
 ```
 
+or [Download the BibTeX file](./assets/DanischKrumbiegel2021.bibtex).
+
 # Installation
 
 Please consider using the backends directly. As explained in the documentation, they re-export all of Makie's functionality.
 So, instead of installing Makie, just install e.g. GLMakie directly:
+
 ```julia
 julia>]
 pkg> add GLMakie
-pkg> test GLMakie
 ```
 
+You may check the installed version with:
+
+```julia
+]st GLMakie
+```
+
+Test the package:
+
+```julia
+using GLMakie
+] test GLMakie
```

Comment:
I think we should just remove that.. I personally wouldn't recommend running the tests on the local machine, since platform problems will immediately error without tests and if there are no platform specific problems, all versions should come tested by the CI.

Reply:
agree, I never test the packages 😄, straight to using them.  Done!